test/unit/metrics/sample_dashboards/bc-2p-3f.json

```diff
@@ -450,19 +450,19 @@
                 },
                 "fallout_rate": {
                     "bins": [
-                        0.5833333333333333,
+                        0.5833333333333334,
                         0.8333333333333334,
                         0.2727272727272727,
                         0.46153846153846156
                     ],
-                    "global": 0.5416666666666667
+                    "global": 0.5416666666666666
                 },
                 "miss_rate": {
                     "bins": [
-                        0.6470588235294117,
-                        0.5833333333333333,
+                        0.6470588235294118,
+                        0.5833333333333334,
                         0.25,
-                        0.4545454545454546
+                        0.45454545454545453
```

Comment:
It would appear that I unwisely assumed I didn't need `pytest.approx()` in a few tests....

Reply:
I looked into whether I can use pytest.approx(), but it doesn't seem work with deep data structures, so I just changed the numbers. Maybe there's a simple tweak--that involves flattening json or invoking an approximate comparison on deep structures--but I'm going to leave that for a future PR.

Reply:
Something like
```
np.isclose(A, B).all()
```
may work